doc/source/indexing.rst

```diff
@@ -500,6 +500,62 @@ A list of indexers where any element is out of bounds will raise an
 
 .. _indexing.basics.partial_setting:
 
+Selecting Random Samples
+------------------------
+.. versionadded::0.16.1
+
+A random selection of rows from a Series, DataFrame, or Panel with the ``.sample()`` method. 
+The method accepts a specific number of rows to return, or a fraction of rows. 
+
+.. ipython :: python
+
+   s = Series([0,1,2,3,4,5])
+
+   # When no arguments are passed, returns 5 rows like .head() or .tail()
+   s.sample()
+   
+   # One may specify either a number of rows:
+   s.sample(n = 3)
+   
+   # Or a fraction of the rows:
+   s.sample(frac = 0.5)
+
+By default, ``sample`` will return each row at most once, but one can also sample with replacment
+using the ``replace`` option:
+
+.. ipython :: python
+
+   s = Series([0,1,2,3,4,5])
+		
+   # Without replacement (default):
+   s.sample(n = 6, replace = False)
+	
+   # With replacement:
+   s.sample(n = 6, replace = True)
+
+
+By default, each row has an equal probability of being selected, but if you want rows
+to have different probabilities, you can pass ``sample`` sampling weights. This weights
```

Comment:
I think u mean to say weights here (as that is the argument name) (or maybe just need to put back ticks around weights)